DevGuideExamples/DCPS/Messenger.minimal/Publisher.cpp

```diff
@@ -55,6 +55,13 @@ ACE_TMAIN(int argc, ACE_TCHAR *argv[])
         msg_count = ACE_OS::atoi(argv[1]);
       }
 
+      if (msg_count < 0 || msg_count > 100)
+      {
```

Comment:
Update brace placement for consistency (more important in DevGuideExamples than in tests).